### PR TITLE
feat: C2-9049: Implement zoom restriction in SDA

### PR DESCRIPTION
**Pull request by parthprajapati05**
- **Note: Work in progress for the UTs.**
## Description - zoom restriction in SDA
Add Zoom restriction as per the following

- Set the maximum zoom-in % to 150% (goes from 100% < 110% < 125% < 150%)

- Set the maximum zoom-out % to 70% (goes from 100% > 90% > 80% > 70%)

[C2-9049](https://perzoinc.atlassian.net/browse/C2-9049)

### CODE REVIEW

**Review by utsavshah9 — APPROVED** [review]

**Review by NavyaGopalakrishna — APPROVED** [review]

File: src/app/main-api-handler.ts [hunk 1]
```diff
@@ -251,6 +251,11 @@ ipcMain.on(
       case apiCmds.setIsMana:
         if (typeof arg.isMana === 'boolean') {
           windowHandler.isMana = arg.isMana;
+          // Update App Menu
+          const appMenu = windowHandler.appMenu;
+          if (appMenu && windowHandler.isMana) {
+            appMenu.buildMenu();
+          }
```
Comment (VishwasShashidhar) [inline src/app/main-api-handler.ts:258]: @KiranNiranjan Do we need to rebuild the menu here since we already do it on `did-finish-load`?

**Review by prakashSolanki0702 — APPROVED** [review]

File: src/app/main-api-handler.ts (hunk 1 above)
Comment (KiranNiranjan) [reply src/app/main-api-handler.ts:258]: Not required unless we add or remove any menu items after the app is loaded

Comment (parthprajapati05) [reply src/app/main-api-handler.ts:258]: @VishwasShashidhar  @KiranNiranjan  We are changing the menu action (assign zoomIn/Out method to menu) when `isMana` is set to true.

Comment (VishwasShashidhar) [reply src/app/main-api-handler.ts:258]: Sounds good @parthprajapati05. I'll merge the PR once you have unit tests ready.

**Comment by sandhyarajput** [comment]
ok to test

**Review by VishwasShashidhar — APPROVED** [review]

**Pull request merged**